linkwatch/endpoints.py

```diff
@@ -4,8 +4,8 @@
 import logging
 import datetime, pytz
 import constants
+from settings import *
```

Comment:
@paraschivionut BTW, this is bad practice to pollute the local namespace with unknown symbols. I would go for importing settings as an object and using keys from it.